src/services/PermissionServiceNative.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  request,
+  check,
+  requestNotifications,
+  PERMISSIONS,
+  Permission
+} from 'react-native-permissions'
+import { Platform } from 'react-native'
+import { OS_TYPES } from '../constants'
+// type used in usage examples
+type PermissionStatus =
+  | 'unavailable'
+  | 'denied'
+  | 'limited'
+  | 'granted'
+  | 'blocked'
+  | null
+
+export async function askForNotificationsPermission(): Promise<PermissionStatus> {
+  requestNotifications(['alert', 'sound']).then(({ status, settings }) => {
+    return status
+  })
+  return null
+}
+
+export async function askForPermission(
+  permissionToGet: any
```

Comment:
Please don't use `any`!!!

You could say it's a string
```suggestion
  permission: string
```

or even better:
```suggestion
  permission: 'CAMERA' | 'PHOTO_LIBRARY'
```